operator/src/main/java/com/github/streamshub/console/api/v1alpha1/spec/security/Audit.java

```diff
@@ -0,0 +1,38 @@
+package com.github.streamshub.console.api.v1alpha1.spec.security;
+
+import java.util.Locale;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import io.fabric8.generator.annotation.Required;
+import io.sundr.builder.annotations.Buildable;
+
+@Buildable(editableEnabled = false)
+public class Audit extends Rule {
+
+    @Required
+    Decision decision;
+
+    public Decision getDecision() {
+        return decision;
+    }
+
+    public void setDecision(Decision decision) {
+        this.decision = decision;
+    }
+
+    public enum Decision {
+        ALLOWED,
+        DENIED,
+        ALL;
+
+        @JsonCreator
+        public static Decision forValue(String value) {
+            if ("*".equals(value)) {
```

Comment:
Do we need to guard null?

Reply:
> Do we need to guard null?

Kube rejects the CR with:
> Error "Required value" for field "spec.security.audit[0].decision".

Similarly, attempts to use a null for an entry in the `privileges` array results in:
> Error "Invalid value: "null": spec.security.audit[0].privileges[1] in body must be of type string: "null"" for field "spec.security.audit[0].privileges[1]".